web-app/django/VIM/apps/instruments/views/instrument_detail.py

```diff
@@ -0,0 +1,45 @@
+from django.views.generic import DetailView
+from VIM.apps.instruments.models import Instrument, Language
+
+
+class InstrumentDetail(DetailView):
+    """
+    Displays details of a specific instrument.
+    """
+
+    model = Instrument
+    template_name = "instruments/detail.html"
+    context_object_name = "instrument"
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        instrument = self.get_object()

```

Comment:
`context["instrument"]` will already have the `instrument` you want, so you don't need to `get_object` (this leads to a duplicate query).